Fixes #82 Case Insensitive Searching

ThuyNguyen97 opened the pull request:
Fix issue #82 'cont' not working on Oracle

seanfcarroll commented:
This looks great @ThuyNguyen97 !

Could you please:
- Add some supporting tests 
- Update the README.md to add a description of the `icont` and `inot` methods

Thanks very much 😄

ThuyNguyen97 commented:
I have:

-  updated `attr_value_for_attribute` method
- rename `icont` -> `i_cont`, `inot_cont` -> `not_i_cont`
- added some rspec for `i_cont` and `not_i_cont`
- update README.md

I will edit wiki to description of the `case_insensitive` option later
Pls check again :smile:

seanfcarroll commented:
Thank you for the quick response @ThuyNguyen97
